src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -159,7 +158,7 @@ public ProcessResult updateCase(
         } catch (UnprocessableEntity exception) {
             ClientServiceErrorResponse errorResponse = serviceResponseParser.parseResponseBody(exception);
             return new ProcessResult(errorResponse.warnings, errorResponse.errors);
-        } catch (Conflict exception) {
+        } catch (FeignException.Conflict exception) {
```

Comment:
I've looked in the code and it turns out we don't need this `catch` here at all, because 409 is handled in `updateCaseInCcd`. If here we get any other 409, that should be treated as any error.